addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -1527,12 +1537,16 @@ var SnippetsMenu = Widget.extend({
                 // ... if no editors are to be enabled, look if any have been
                 // enabled previously by a click
                 if (!editorToEnable) {
-                     editorToEnable = this.snippetEditors.find(editor => editor.isSticky());
-                     previewMode = false;
+                    editorToEnable = this.snippetEditors.find(editor => editor.isSticky());
+                    previewMode = false;
```

Comment:
Speaking about previewMode, ideally we should still have the preview overlay working as before. So if you hover the column option, the blue overlay is shown around the column and if you hover the image option, the blue overlay is shown around the image.